src/bitmessagekivy/mpybit.py

```diff
@@ -206,6 +206,56 @@ def load_screen_callback(self, instance, dt=0):
             except Exception as e:
                 self.root.ids.id_trash.children[0].children[1].active = False
 
+    @staticmethod
+    def current_address_label(current_add_label=None, current_addr=None):
+        """Getting current address labels"""
+        addresses = [addr for addr in config.addresses()
+                     if config.getboolean(str(addr), 'enabled')]
+        if addresses:
+            if current_add_label:
+                first_name = current_add_label
+                addr = current_addr
+            else:
+                addr = addresses[0]
+                first_name = config.get(addr, 'label')
+                if not config.getboolean(addr, 'enabled'):
+                    return ''
+            f_name = first_name.split()
+            truncate_string = '...'
+            f_name_max_length = 15
+            label = f_name[0][:14].capitalize() + truncate_string if len(
+                f_name[0]) > f_name_max_length else f_name[0].capitalize()
+            address = ' (' + addr + ')'
+            return label + address
+        return ''
+
+    def getDefaultAccData(self, instance):
+        """Getting Default Account Data"""
+        if self.identity_list:
+            self.kivy_state_obj.association = first_addr = self.identity_list[0]
+            return first_addr
+        return 'Select Address'
+
+    def getCurrentAccountData(self, text):
+        """Get Current Address Account Data"""
+        if text != '':
+            if os.path.exists(os.path.join(
+                    self.image_dir, 'default_identicon', '{}.png'.format(text))
+            ):
+                self.load_selected_Image(text)
+            else:
+                self.set_identicon(text)
+                self.root.ids.content_drawer.ids.reset_image.opacity = 0
+                self.root.ids.content_drawer.ids.reset_image.disabled = True
+            address_label = self.current_address_label(
```

Comment:
rewrite this not to use `current_address_label`

Reply:
replaced with `get_current_label`

Reply:
`address_label = format_address_and_label(text)`
no need to use `config.get`